packages/aws-cdk/lib/api/provisioner.ts

```diff
@@ -0,0 +1,44 @@
+import cxapi = require('@aws-cdk/cx-api');
+import { debug } from '../logging';
+import { deserializeStructure } from '../serialize';
+import { Mode } from './aws-auth/credentials';
+import { SDK } from './util/sdk';
+
+export type Template = { [key: string]: any };
+
+/**
+ * Interface for provisioners
+ *
+ * Provisioners apply templates to the cloud infrastructure.
+ */
+export interface IProvisioner {
```

Comment:
Can we use `DeploymentTask` to align with the terminology from #1893?

Reply:
Yes on `Deployment`, not sure `Task` is the right word. `DeploymentTarget`?

Reply:
I agree that Task is not good. Maybe `DeploymentProvider`?